### ISSUE TITLE
Running button to « play » only specific groups.

### ISSUE THREAD

**Issue by yovsac** [chrisgoringe/cg-controller#455]
It would be great to have a running button similar to rgthrees to run only the workflows in a specific group. This is interesting if you have different tools within a workflow like LLMs were you just want to have a query for a prompt that will help you but it’s not necessarily part of the main workflow. If that makes sense.

**Comment by JorgeR81** [comment]
If this is an rgthree feature ( https://github.com/chrisgoringe/cg-controller/issues/454#issuecomment-2541394871 ), we could have, a button in each controller window that calls directly for this rgthree function.

We have plenty of space in the bottom header.

We could use the same icon.

![ic1](https://github.com/user-attachments/assets/d21fd7bf-1fe9-474a-abb2-e7604af3f4f2)

![r2](https://github.com/user-attachments/assets/da59ec5d-b07e-4ead-be6a-eb36cd390e33)

---

But maybe we could have an user option to enable this first, like rgthree. 

![r1](https://github.com/user-attachments/assets/217709a8-bd45-46c3-926e-72e146c45329)

**Comment by yovsac** [comment]
Yes that is it. when you select a group in the controller have that play button next to it. 

Yeah it could be activated like rgthree or have it by default.

**Comment by chrisgoringe** [comment]
I’ll take a look at the rgthree version. My first preference would be to
hook into their code rather than reinventing.
